postgres/defaults.yaml

```diff
@@ -41,3 +41,7 @@ postgres:
   databases: {}
   schemas: {}
   extensions: {}
+
+  linux:
+    #Alternatives system are disabled by a 'altpriority=0' pillar.
+    altpriority: 30
```

Comment:
Since for now the formula is able to provision the alternatives only for RedHat systems when PG comes from upstream repo, I think the default value across all supported distros must be empty:
```
linux:
  # Alternatives are applicable only for some OS
  altpriority:
```

Please set actual value in `osmap.yaml` file near `bin_dir` and `*_bins` parameters.